src/char_table.h

```diff
@@ -32,7 +35,9 @@ struct CharTable
       {
          return(0);
       }
-      if (ch < (int)ARRAY_SIZE(chars))
+      //if (ch < (int)ARRAY_SIZE(chars))
+      // Coverity: CID 76238 (#1 of 1): Out-of-bounds read (OVERRUN)
+      if (ch < CharTable_Length)
```

Comment:
There are 29 matches of ARRAY_SIZE usage in the code, and this one looks correct to me. Why is Coverity complaining only about this one, and why is switching to a #define at all helpful?

This looks like the wrong fix to me.